index.bs

```diff
@@ -933,6 +935,35 @@ authorizing an authenticator with which to complete the operation.
 </div>
 
 
+### Platform Authenticator Availability - PublicKeyCredential's `isPlatformAuthenticatorAvailable()` method ### {#isPlatformAuthenticatorAvailable}
+
+<div link-for-hint="WebAuthentication/isPlatformAuthenticatorAvailable">
+
+[=[RPS]=] use this method to determine whether they can create a new credential using a [=platform authenticator=].
+Upon invocation, the [=client=] employs a platform-specific procedure to discover available [=platform authenticators=].
+If successful, the [=client=] then assesses whether the user is willing to create a credential using one of the available [=platform authenticators=].
+This assessment may include various factors, such as:
+    - Whether the user is running in private or incognito mode.
+    - Whether the user has configured the [=client=] to not create such credentials.
+    - Whether the user has previously expressed an unwillingness to create a new credential for this [=[RP]=],
+        either through configuration or by declining a user interface prompt.
+    - The user's explicitly stated intentions, determined through user interaction.
+If this assessment is affirmative, the promise is resolved with the value of <i>True</i>.
+Otherwise, the promise is resolved with the value of <i>False</i>.
```

Comment:
The delay shouldn't be user visible in any sense, since the rp's action upon receiving "false" is to do nothing. A "true" response, which does require some user action or interaction, would be conveyed immediately. The concept of the delay is just to hide which of the three failure cases is resolving the promise. 
(I don't disagree that it might cause possible battery drain.)

I guess my real question might be - have we agreed to give up on the idea that we shouldn't try to prevent fingerprinting?